docs/api.rst

```diff
@@ -47,6 +47,12 @@ SELIC
 .. autoclass:: calculadora_do_cidadao.Selic
    :members:
 
+Família CPI
```

Comment:
This is (yet) not a family, but a single adapter, right? Can we have its name here (that is to say, _CPIAUCSL_, as in SELIC and IGPM-P) instead of the name of a family of a single member?

Reply:
Done